taskcluster/ci/github-release/kind.yml

```
@@ -0,0 +1,39 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: ac_taskgraph.loader.all_dep:loader
+
+transforms:
+    - ac_taskgraph.transforms.github_release:transforms
```

Bug: We should add the `all_dep` transform.

```suggestion
    - ac_taskgraph.transforms.all_dep:transforms
    - ac_taskgraph.transforms.github_release:transforms
```

This will solve the bug you saw where `github-release` aren't present in the release graph. What happens is: 
 * both the nightly and the release graph use `build-type` to know what task to include or not. E.g.: 

https://github.com/mozilla-mobile/android-components/blob/44987b3668e20a146b20749a8584e88d572fcfc5/taskcluster/ac_taskgraph/target_tasks.py#L18-L23

 * the `all_dep` transform copies attributes from the parent tasks. `build-type` is an attribute. 

https://github.com/mozilla-mobile/android-components/blob/fc16a83dcd7da38136d2be9e18ecd523b4976a0d/taskcluster/ac_taskgraph/transforms/all_dep.py#L23